build/yamls/antrea-aks.yml

```diff
@@ -3393,6 +3393,27 @@ rules:
   - get
   - watch
   - list
+- apiGroups:
```

Comment:
make manifest (after updating base rbac file) updated all the antrea-*.yml file. But, not sure if we really wanted to enable secondarynetwork config on all the variants?. if not, I can remove these changes.